vscode/src/services/AuthProvider.ts

```diff
@@ -108,7 +108,11 @@ export class AuthProvider {
                 if (!authStatus?.isLoggedIn) {
                     const newToken = await showAccessTokenInputBox(item.uri)
                     const authStatusFromToken = await this.auth(selectedEndpoint, newToken || null)
-                    this.showIsLoggedIn(authStatusFromToken?.authStatus || null)
+                    if (authStatusFromToken?.isLoggedIn) {
```

Comment:
```suggestion
                    this.showIsLoggedIn(authStatusFromToken?.authStatus || null)
```
I haven't gone through the code in awhile but is the if/else statement still needed if we move `showAuthFailureMessage()` into `this.showIsLoggedIn()`  for `null` or failed AuthStatus?